bindings/python/setup.py

```diff
@@ -184,74 +240,152 @@ def initialize_options(self):
             pass
 
         if os.name == "nt":
-            self.libtorrent_link = 'static'
+            self.toolset = get_msvc_toolset()
```

Comment:
I started doubting whether `get_msvc_toolset()` is necessary. It always returns `msvc-14.1` or `msvc` on supported python versions.

I couldn't determine from the referenced wiki page whether there was a meaningful difference among `14.x` versions for python extensions.

Also my CI for #5948 broke in windows when I passed `b2 toolset=msvc-14.1`. It worked when I left `toolset` unconfigured. I don't remember the error now. `msvc-14.2` is installed on the CI image.

If autodetection is very broken on older supported boost versions, can we get away with a default of `msvc` instead of `msvc-14.1`?

Reply:
yeah, My understanding is that with msvc committing to longer term ABI stability, new versions of python no longer has this additional burden.

Reply:
I think it's fine to remove in later branches

Reply:
Perhaps we could strip down the function to remove checks for older unsupported pythons, and document that it's a deprecated default

Reply:
yes, that makes sense. in future versions, like `RC_2_0` or later